common/src/main/resources/assets/wynntils/lang/en_us.json

```
@@ -1960,13 +1960,17 @@
   "service.wynntils.itemFilter.stat.emeraldValue.description": "Emerald value of the item",
   "service.wynntils.itemFilter.stat.gearRestriction.description": "Gear restriction of the item",
   "service.wynntils.itemFilter.stat.gearType.description": "Gear type of the item (e.g. helmet)",
+  "service.wynntils.itemFilter.stat.health.description": "Health of the item",
```

```suggestion
  "service.wynntils.itemFilter.stat.health.description": "The item's base health value",
```

To prevent confusion with +health bonus stats